core/modules/system/css/system.theme.css

```diff
@@ -585,6 +585,19 @@ tr:hover del.diffchange {
 }
 
 /**
+ * Hero blocks.
+ */
+ .hero-block {
+   background-size: cover;
+   background-repeat: no-repeat;
+   background-position: top center;
+   padding: 50px 100px;
+   text-align: center;
+   color: #ffffff;
```

Comment:
If we are going to set the font-color, we should set a background color as well. Otherwise with no theme-specific CSS and no image you'll end up with white-on-white. However my preference might be to just omit any color entirely here and just leave that entirely to the theme.